src/pages/toolkit/bpmn-js/walkthrough/walkthrough.md

```diff
@@ -75,41 +75,42 @@ The included script makes the viewer available via the `BpmnJS` variable. We may
 ```html
 <script>
   // the diagram you are going to display
-  var bpmnXML;
+  const bpmnXML;
 
   // BpmnJS is the BPMN viewer instance
-  var viewer = new BpmnJS({ container: '#canvas' });
+  const viewer = new BpmnJS({ container: '#canvas' });
 
   // import a BPMN 2.0 diagram
-  viewer.importXML(bpmnXML, function(err) {
-    if (err) {
-      // import failed :-(
-    } else {
-      // we did well!
-
-      var canvas = viewer.get('canvas');
-      canvas.zoom('fit-viewport');
-    }
-  });
+  try {
+    // we did well!
+    await viewer.importXML(bpmnXML);
+    viewer.get('canvas').zoom('fit-viewport');
+
+  } catch (err) {
+    // import failed :-(
+  }
 </script>
 ```
 
-The snippet uses the [`Viewer#importXML`](https://github.com/bpmn-io/bpmn-js/blob/master/lib/Viewer.js#L147) API to display a pre-loaded BPMN 2.0 diagram. Importing a diagram is asynchronous and, once finished, the viewer notifies us via a callback about the results.
+The snippet uses the [`Viewer#importXML`](https://github.com/bpmn-io/bpmn-js/blob/master/lib/BaseViewer.js#L109) API to display a pre-loaded BPMN 2.0 diagram. Importing a diagram is asynchronous and, once finished, the viewer notifies us via a callback about the results.
 
-After import, we may access various diagram services via [`Viewer#get`](https://github.com/bpmn-io/bpmn-js/blob/master/lib/Viewer.js#L246). In the snippet above, we interact with the [`Canvas`](https://github.com/bpmn-io/diagram-js/blob/master/lib/core/Canvas.js#L77) to fit the diagram to the currently available viewport size.
+After import, we may access various diagram services via [`Viewer#get`](https://github.com/bpmn-io/bpmn-js/blob/master/lib/BaseViewer.js#L486). In the snippet above, we interact with the [`Canvas`](https://github.com/bpmn-io/diagram-js/blob/master/lib/core/Canvas.js) to fit the diagram to the currently available viewport size.
 
 Often times it is more practical to load the BPMN 2.0 diagram dynamically via AJAX.
 This can be accomplished using plain JavaScript (as seen below) or via utility libraries such as [jQuery](https://api.jquery.com/jQuery.get), which provide more convenient APIs.
 
 ```html
 <script>
-  var xhr = new XMLHttpRequest();
```

Comment:
We could make this one use [modern `fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), too.